cmd/util-db/db/info.go

```
@@ -165,6 +176,16 @@ func printCount(cfg *utils.Config, log logger.Logger) error {
 		}
 	}
 
+	// print exception count
+	if dbComponent == dbcomponent.Exception || dbComponent == dbcomponent.All {
+		count, err := utildb.GetExceptionCount(cfg, base)
+		if err != nil {
+			log.Warningf("cannot print exception count; %v", err)
```

return err instead of loggin?